### DISCUSSION TITLE
Data feed

### DISCUSSION THREAD

**Discussion by alberk8** [elsa-workflows/elsa-core#4139] [Q&A]
Hi,
If I have a SNMP feed and on certain critical event I want to trigger an alert to a phone app. In addition if the same event happens within a duration let say 10minutes from the previous, then don't raise the alert.  Is this possible with Elsa? Will it be able to handle large inflow of data.

**Comment by sfmskywalker** [comment]
Certainly, that is achievable with Elsa. A viable strategy would be to set up the workflow as a singleton. This ensures that while a workflow instance is active, another instance won't initiate simultaneously. When a specific trigger within this workflow is activated, it can dispatch an alert. To prevent frequent alerts for the same event, the workflow can incorporate a Delay activity that pauses it for a 10-minute duration. Once this interval concludes, the workflow terminates, making it ready to process subsequent events.

As for managing a substantial data influx, Elsa 3 is tailored to support ETL processes efficiently. However, it's essential to note that its capacity to handle massive data volumes hasn't been rigorously evaluated, given that it's still in an intensive development phase.
